lib/server.js

```diff
@@ -90,7 +91,18 @@ function create(log, error, config, routes, db, translator) {
     }
   }
 
+  var shouldCacheDevices = config.devicesCache.enabled
```

Comment:
need to rename this, seems like we always cache, this boolean is about redis usage